flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/catalog/CatalogTableITCase.scala

```diff
@@ -440,6 +440,103 @@ class CatalogTableITCase(isStreaming: Boolean) {
     execJob("testJob")
     assertEquals(TestCollectionTableFactory.RESULT.sorted, sourceData.sorted)
   }
+
```

Comment:
add a test for drop not exist table but with tolerance.

Reply:
Already added